test/e2e/libpod_suite_test.go

```diff
@@ -363,14 +363,14 @@ func (p *PodmanTest) RestoreAllArtifacts() error {
 	return nil
 }
 
-//RunSleepContainer runs a simple container in the background that
-// sleeps.  If the name passed != "", it will have a name
-func (p *PodmanTest) RunSleepContainer(name string) *PodmanSession {
+//RunTopContainer runs a simple container in the background that
+// runs top.  If the name passed != "", it will have a name
+func (p *PodmanTest) RunTopContainer(name string) *PodmanSession {
 	var podmanArgs = []string{"run"}
 	if name != "" {
 		podmanArgs = append(podmanArgs, "--name", name)
 	}
-	podmanArgs = append(podmanArgs, "-d", ALPINE, "sleep", "90")
+	podmanArgs = append(podmanArgs, "-d", ALPINE, "top")
```

Comment:
Do we still have a least one test path that checks to see that a container running sleep can be stopped?  Even if it takes 10 seconds?

Reply:
A sleep container sounds like a good way of testing the timeout until SIGKILL in `podman stop`